### ISSUE TITLE
QuiteRSS crashing

### ISSUE THREAD

**Issue by howardneil** [QuiteRSS/quiterss#1576]
Starting today, QuiteRSS keeps crashing (dispaying the notice "not responding".

**Comment by Rex000** [comment]
Yes it's been having this issue for a while now. The most annoying thing is when it silently crashes in the background without even an error message.

**Comment by howardneil** [comment]
I have decided that QuiteRSS is no longer being supported and I have now changed to Feedly. It is superb and I wish I had changed before.

**Comment by jay9883bea** [comment]
I have been having the same problem for a while too. I haven't been able to track it down to a particular cause, but it usually happens when updating, so I assume that there is something in one of my feeds that causes it. It crashed repeatedly this morning whilst updating, but I managed to keep it open long enough to exports my current feeds list to take elsewhere.

I have a looked through the debug log (AppData\Local\QuiteRss\QuiteRss\debug.log) and the only critical error related to Google Analytics

`12.06.2023 08:31:41.446 CRITICAL: replyFinished error:  "Host www.google-analytics.com not found"`

I don't think that the application crashes immediately after this error. However, this error only appears in the log infrequently and seems to correlate when I have had crashes. There are no other critical errors that might give any clues.

**Comment by TheGimkitGuy4729** [comment]
Yea I know

**Comment by birdie-github** [comment]
An active fork with full Qt6 support is here: https://github.com/birdie-github/notquiterss

You're welcome to try it.
